util/create_report_html.py

```diff
@@ -110,9 +101,8 @@ def maybe_get_link(cell, context):
 }
 
 other_prefixes = {
-    'dc': 'http://purl.org/dc/terms/',
     'dcterms': 'http://purl.org/dc/terms/',
-    'dc11': 'http://purl.org/dc/elements/1.1/',
+    'dc': 'http://purl.org/dc/elements/1.1/',
```

Comment:
Why this change?

Reply:
renaming to the right name

Reply:
"right" how? `dce` is elements, `dcterms` is terms. `dc` sounds a bit ambiguous to me.. Anyways, you can make that call without my approval!

Reply:
Actually, I've never seen xml/rdf file using dce, always dc.

Reply:
I'll change dcterms to terms.